part.go

```diff
@@ -28,11 +29,11 @@ const (
 // Part represents a node in the MIME multipart tree.  The Content-Type, Disposition and File Name
 // are parsed out of the header for easier access.
 type Part struct {
-	PartID      string               // PartID labels this part's position within the tree.
-	Parent      *Part                // Parent of this part (can be nil.)
-	FirstChild  *Part                // FirstChild is the top most child of this part.
-	NextSibling *Part                // NextSibling of this part.
-	Header      textproto.MIMEHeader // Header for this part.
+	PartID      string           // PartID labels this part's position within the tree.
+	Parent      *Part            // Parent of this part (can be nil.)
+	FirstChild  *Part            // FirstChild is the top most child of this part.
+	NextSibling *Part            // NextSibling of this part.
+	Header      nettp.MIMEHeader // Header for this part.
```

Comment:
I think this would be cleaner for people looking at docs if you rename the import of `.../internal/textproto` instead of `net/textproto`.